Readme.md

```diff
@@ -46,15 +46,95 @@ If you'd like to build your own VFS for Git Windows installer:
 
 The installer can now be found at `C:\Repos\VFSForGit\BuildOutput\GVFS.Installer\bin\x64\[Debug|Release]\SetupGVFS.<version>.exe`
 
+## Building VFS for Git on Mac
+
+Note that VFS for Git on Mac is under active development.
+
+* Ensure you have Xcode installed, have accepted the terms of use, and have launched Xcode at least once.
+
+* Disable the "System Integrity Protection" (for loading unsigned Kexts) by booting into recovery mode (`[Win/⌘] + R` while booting).
+  Once booted into recovery mode open Utilities -> Terminal to launch a terminal. Enter:
+
+  ```
+  csrutil disable
+  ```
+  to disable SIP. Then click the Apple logo in the top left and restart.
+
+* Install [Visual Studio for Mac ](https://visualstudio.microsoft.com/vs/mac). (This will also install the `dotnet` CLI).
+
+* If you still do not have the `dotnet` cli `>= v2.1.300` installed [manually install it]. You can check what version you have with `dotnet --version`.(https://www.microsoft.com/net/download/dotnet-core/2.1)
+
+* You will need to manage and sign your own certificate. [Apple Developer Cert Docs](https://developer.apple.com/documentation/security/certificate_key_and_trust_services/certificates).
+
+* Create a `VFSForGit` directory and Clone VFSForGit into a directory called `src` inside it:
+  ```
+  mkdir VFSForGit && cd VFSForGit && git clone https://github.com/Microsoft/VFSForGit.git src && cd src
+  ```
+
+* From the src directory run
+
+  ```
+  Scripts/Mac/BuildGVFSForMac.sh [Release]
+  ```
+
+  _Troubleshooting if this fails_
+
+  If you get
+  ```
+  xcodebuild: error: SDK "macosx10.13" cannot be located.
+  ```
+  You may have the "XCode Command Line Tools" installed (helpfully by Mac OS) instead of full XCode.
+  Make sure
+  ```
+  xcode-select -p
+  ```
+
+  shows `/Applications/Xcode.app/Contents/Developer`. If it does not, install Xcode and then launch it (you can close it afterwards.)
+
+* Prep your machine to use VFS for Git. The following are all done by the script below.wwwww
+  * install Homebrew
+  * install and setup the git credential manager (with `brew`)
+  * install/update Java (with `brew`)
+  * install a VFS for Git aware version of Git
+
+  ```
+  Scripts/Mac/PrepFunctionalTests.sh
+  ```
+
+* Now you have to load the ProjFS Kext.
+
+  ```
+  ProjFS.Mac/Scripts/LoadPrjFSKext.sh [Release]
+  ```
+
+* Add your built gvfs program to your path. A simple way to do that is by adding `Path/to/VFS/Publish` to your path.
+
+  Confirm you have it by running
+
+  ```
+  command -v gvfs
+  ```
+
+  You should see a path to the gvfs executable.
+
+* Try cloning a VFS for Git enabled repository!
+
+  ```
+  gvfs clone URL_TO_REPOSITORY --local-cache-path ~/.gvfsCache
+  ```
+
+  Note the current use of `--local-cache-path`. That argument prevents VFS for Git from running into a permissions error trying to put the cache at the root of your Mac's hard-drive because porting of this feature is still in progress.

```

Comment:
Style comment.

I found this:

> Note the current use of `--local-cache-path`. That argument prevents VFS for Git from running into a permissions error trying to put the cache at the root of your Mac's hard-drive because porting of this feature is still in progress.

A bit tough to follow.  Consider splitting up the second sentence and adding a few more details:

> Note the current use of `--local-cache-path`. Without this argument VFS for Git will encounter a permissions error when it attempts to create its cache at the root of your hard-drive. Automatic picking of the cache path has not yet been ported to VFS for Git on Mac.